package.json

```diff
@@ -54,15 +54,15 @@
     "@bitcoin-design/bitcoin-icons-react": "^0.1.9",
     "@heroicons/react": "^1.0.6",
     "axios": "^0.27.2",
-    "event-source-polyfill": "1.0.25",
+    "event-source-polyfill": "1.0.31",
```

Comment:
don't want to update that; see https://github.com/fusion44/blitz_api/issues/144

Reply:
Forgot about that